CHANGES.md

```diff
@@ -5,7 +5,7 @@
 * Stopped the registry api from timing out and returning an error code when trimming by source tag / id - now returns 202 if it takes too long.
 * Added route for `/pages/*` requests so that `magda-web-server` won't response `Cannot GET /page/*`
 * Added format sleuther
-* Set `kubernetes-client` (required by magda-admin-api) version to `3.17.2` to sovle the travis build issue* Added ability to get records from the registry by the value of their aspects.* Added ability to get records from the registry by the value of their aspects.* Added access control layer to Authorization APIs: All `private` APIs (uri starts with /private/) can only be accessed by Admin users.
+* Set `kubernetes-client` (required by magda-admin-api) version to `3.17.2` to sovle the travis build issue* Added ability to get records from the registry by the value of their aspects.* Added ability to get records from the registry by the value of their aspects.\* Added access control layer to Authorization APIs: All `private` APIs (uri starts with /private/) can only be accessed by Admin users.
```

Comment:
Not new, but looks like there are three bullets on one line.  Also, the `\`, which is new, is extraneous.